src/RadioButtonList/widget/template/RadioButtonList.html

```diff
@@ -1,4 +1,4 @@
-<div class="RadioButtonList form-group">
+<div class="RadioButtonList mx-radiobuttons form-group" data-dojo-attach-point="radioButtonContainer">
     <label class="control-label" data-dojo-attach-point="radioButtonLabel"></label>

```

Comment:
```suggestion
    <label class="control-label" id="{this.uniqueid}-label" data-dojo-attach-point="radioButtonLabel"></label>
```
not sure if this would work :D
https://apidocs.mendix.com/8/client/mxui_widget__WidgetBase.html
https://dojotoolkit.org/documentation/tutorials/1.9/templated/